### ISSUE TITLE
Major Issue on UI

### ISSUE THREAD

**Issue by sh-biswas** [MLH-Fellowship/Student-Planner#7]
The UI is not rendering in Express and Node.js. I am trying to integrate the bootstrap templates, but it seems they are not working with the templating engine. We might need to do a major redesign, but I am not sure yet. This is absolutely urgent, as we have still not created a solid UI and connected it with the backend. I am currently trying to figure out how to make bootstrap work with Express. Here is what the homepage renders as:
<img width="1438" alt="Screen Shot 2020-10-12 at 11 21 41 PM" src="https://user-images.githubusercontent.com/51776663/95822992-bcd23600-0ce1-11eb-86dd-17b6938cedb6.png">

**Comment by V2dha** [comment]
Did you check the file directory because with this it seems like it's not getting the css and bootsrap files.

**Comment by V2dha** [comment]
Can you push the changes in a separate branch or in the same branch so we can debug it?

**Comment by sh-biswas** [comment]
I have pushed in the relevant changes that show the issue, and I have also added a README that gives details about how to see the issue in your own browser. I have already followed the instructions in the stackoverflow link, but the same issue is still appearing.

**Comment by V2dha** [comment]
https://github.com/MLH-Fellowship/Student-Planner/blob/913759da72486fb9cd789a60339b6b0a4f98d97e/app.js#L6

I added the above code in the app.js file and it worked on my local server. You also try hopefully now the css and js files are being rendered. I have pushed the changes just pull the changes once and run the local server.

![Screenshot (56)](https://user-images.githubusercontent.com/50369708/95873219-5145a280-0d8d-11eb-904f-c3e32c435649.png)
